### PR TITLE
8328385: Convert java/awt/FileDialog/FileDialogReturnTest test to main

### PR DESCRIPTION
**Pull request by azuev-java**
Convert test to a main method based;
Move test to an apropriate location;
Remove old files;

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328385](https://bugs.openjdk.org/browse/JDK-8328385): Convert java/awt/FileDialog/FileDialogReturnTest test to main (**Bug** - P4)

### Reviewers
 * [Alisen Chung](https://openjdk.org/census#achung) (@alisenchung - Committer)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18363/head:pull/18363` \
`$ git checkout pull/18363`

Update a local copy of the PR: \
`$ git checkout pull/18363` \
`$ git pull https://git.openjdk.org/jdk.git pull/18363/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18363`

View PR using the GUI difftool: \
`$ git pr show -t 18363`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18363.diff">https://git.openjdk.org/jdk/pull/18363.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18363#issuecomment-2005005646)

### CODE REVIEW

File: test/jdk/java/awt/FileDialog/FileDialogReturnTest.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+/*
+ * @test
+ * @bug 6260676
+ * @summary FileDialog.setDirectory() does not work properly, XToolkit
+ * @requires (os.family == "linux")
+ * @library /java/awt/regtesthelpers
+ * @library /test/lib
+ * @build PassFailJFrame
+ * @build jtreg.SkippedException
```
Comment (DamonGuy) [inline test/jdk/java/awt/FileDialog/FileDialogReturnTest.java:46]:
```suggestion
 * @library /java/awt/regtesthelpers /test/lib
 * @build PassFailJFrame jtreg.SkippedException
```

Not positive if this is correct, but I was able to combine the `@library` and `@build` on one line by adding a space between the two.

File: test/jdk/java/awt/FileDialog/FileDialogReturnTest.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+ * @run main/manual FileDialogReturnTest.html
```
Comment (DamonGuy) [inline test/jdk/java/awt/FileDialog/FileDialogReturnTest.java:47]: This still runs on the `.html` file. Does this run correctly for you when testing?

**Review by DamonGuy — CHANGES_REQUESTED** [review]

File: test/jdk/java/awt/FileDialog/FileDialogReturnTest.java (hunk 2 above)
Comment (azuev-java) [reply test/jdk/java/awt/FileDialog/FileDialogReturnTest.java:47]: No, i were fixing other stuff and accidentally pasted the .html there. Will fix that.

File: test/jdk/java/awt/FileDialog/FileDialogReturnTest.java (hunk 1 above)
Comment (azuev-java) [reply test/jdk/java/awt/FileDialog/FileDialogReturnTest.java:46]: I prefer to use separate lines just to make it more readable.

File: test/jdk/java/awt/FileDialog/FileDialogReturnTest.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        PassFailJFrame.builder()
+                .title("File Dialog Return Test Instructions")
+                .instructions(instructions)
+                .rows((int) instructions.lines().count() + 1)
+                .columns(50)
+                .testUI(FileDialogReturnTest::initialize)
+                .build()
+                .awaitAndCheck();
+    }
+}
```
Comment (alisenchung) [inline test/jdk/java/awt/FileDialog/FileDialogReturnTest.java:142]: extra newline

**Review by alisenchung — CHANGES_REQUESTED** [review]

File: test/jdk/java/awt/FileDialog/FileDialogReturnTest.java (hunk 3 above)
Comment (alisenchung) [reply test/jdk/java/awt/FileDialog/FileDialogReturnTest.java:142]: oops, got confused with github formatting, everything else looks good

**Review by alisenchung — APPROVED** [review]

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**